src/Mvc/samples/MvcSandbox/MvcSandbox.csproj

```diff
@@ -7,8 +7,6 @@
   <ItemGroup>
     <Reference Include="Microsoft.AspNetCore.Mvc" />
     <Reference Include="Microsoft.AspNetCore.Mvc.Components.Prerendering" />
-    <Reference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" />
-    <Reference Include="Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation" />
```

Comment:
Runtime compilation doesn't work in the sandbox. It needs PreserveCompilationReferences to be set.